ts/parts-more/Polar.ts

```diff
@@ -792,23 +874,28 @@ if (seriesTypes.column) {
                             startAngleRad;
 
                 } else {
-                    point.shapeType = 'path';
                     start = barX + startAngleRad;
-                    point.shapeArgs = {
-                        d: series.polarArc(
-                            (point.yBottom as any),
-                            (point.plotY as any), start,
-                            start + point.pointWidth
-                        )
-                    };
+
+                    // Changed the way polar columns are drawn in order to make
+                    // it more consistent with the drawing of inverted columns
+                    // (they are using the same function now). Also, it was
+                    // essential to make the animation work correctly (the
+                    // scaling of the group) is replaced by animating each
+                    // element separately.
+                    point.shapeArgs = series.polarArc(
+                        (point.yBottom as any),
+                        (point.plotY as any),
+                        start,
+                        start + point.pointWidth
+                    );
                 }
 
                 // Provided a correct coordinates for the tooltip
                 series.toXY(point);
 
                 if (chart.inverted) {
-                    tooltipPos = xAxis.postTranslate((point as any).rectPlotY,
-                        point.barX + point.pointWidth / 2);
+                    tooltipPos = yAxis.postTranslate((point as any).rectPlotY,
+                        barX + point.pointWidth / 2);
```

Comment:
Is that the reason why tooltips is not rendered in demos?

Reply:
It is caused by this commit: https://github.com/highcharts/highcharts/commit/0b5d8744b7a781130afe3b521dc1da4fc8e946bf.

Reply:
Thanks 👍